src/main/java/BaristaOption.java

```diff
@@ -0,0 +1,22 @@
+import java.util.Arrays;
+
+public enum BaristaOption {
+    REGISTER_MENU(1),
+    SET_STOCK(2),
+    CHANGE_PRICE(3),
+    CHECK_ORDER_HISTORY(4),
+    CHECK_COUPON_HISTORY(5);
+
+    private final int value;
+
+    BaristaOption(int value) {
+        this.value = value;
+    }
+
+    public static BaristaOption getBaristaOption(int value) {
+        return Arrays.asList(BaristaOption.values()).stream()
```

Comment:
```suggestion
	return Arrays.stream(BaristaOption.values())
```

위 코드에서 asList메서드는 Collection의 List로 타입을 변환하는 것이고, 그 후에 stream api를 통해 데이터를 처리하는 것으로 보여지는데 stream api는 array에서도 사용이 가능합니다.

Array -> List -> Stream
Array -> Stream 
의 차이가 있을 것 같네요.!

Reply:
array에서 stream api를 사용할 수 있다는 것을 몰랐네요...
더 공부하겠습니다!
감사합니다.